charts/kf-notebooks/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Global settings
+global:
+  storageClassName: ""
+
+# Common settings
+commonLabels: {}
+commonAnnotations: {}
+nameOverride: ""
+fullnameOverride: ""
+
+# Service settings
+service:
+  enabled: true
+  port: 80
+  type: ClusterIP
+
+# Notebook settings
+notebooks:
```

Comment:
Any specific reason to separate pytorch and vscode? we can have those under the same bucket as well?